### ISSUE TITLE
Vimeo links can have a second path segment which brakes the embed

### ISSUE THREAD

**Issue by SanBen** [jrc9designstudio/statamic-video-embed#24]
Video links that are exposed in certain share dialogs of Vimeo can have two path segments, this breaks the embed.
It seems this happens with private videos that are exposed via a "share" link.

Paste this: ```https://vimeo.com/680825358/fd410b0271```  into the video url field of "statamic-video-embed"

Current behaviour:
breaks because it takes the last path segment even though it should take the first

Expected behaviour:
works because it takes the first path segment

**Comment by jcohlmeyer** [comment]
I do not I will be updating the V2 version of this plugin anymore as V3 has been out for a while, however, if you want to take a stab at it you can send a pull request.

The code would need to be updated to get the video ID in these two places:

https://github.com/jrc9designstudio/statamic-video-embed/blob/master/resources/assets/js/fieldtype.js#L73
https://github.com/jrc9designstudio/statamic-video-embed/blob/master/VideoEmbed.php#L37

**Comment by SanBen** [comment]
> I do not I will be updating the V2 version of this plugin anymore as V3 has been out for a while, however, if you want to take a stab at it you can send a pull request.
> 
> The code would need to be updated to get the video ID in these two places:
> 
> https://github.com/jrc9designstudio/statamic-video-embed/blob/master/resources/assets/js/fieldtype.js#L73 https://github.com/jrc9designstudio/statamic-video-embed/blob/master/VideoEmbed.php#L37

Ok that makes sense, but where is V3? It is not listed in the addons section and I can't seem to find it on GitHub either.

**Comment by jcohlmeyer** [comment]
Yes, I have not yet ported it to V3. As there is now a native video field I was not sure there was much demand for this anymore. However I am planning on working on a new version for Statamic V3, I am just not sure about the timeline for this.
